sintia/sintia.py

```diff
@@ -342,13 +343,18 @@ def format_quote(self, quote: Quote) -> str:
         page_id, *rest = page_ids
         page_info = search_results['query']['pages'][page_id]
         paragraphs = page_info['extract'].split('\n')
-        return await message.channel.send(
-            f'**{page_info["title"]}**\n'
-            f'<{page_info["canonicalurl"]}>'
```

Comment:
nit: inconsistent quotation marks

Reply:
This is because it's an f-string. You can't use `'` inside an already set `f''` string